changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+- bump: patch
+  changes:
+    fixed:
+    - Resolved HDF5 installation error in Python 3.9 Conda environment (Issue #1988).

```

By convention we omit issue numbers from changelogs
```suggestion
    - Add hdf5 to conda environment to avoid installation errors.
```